blog/main.py

```diff
@@ -0,0 +1,66 @@
+import webapp2
+import jinja2
+import os
+import re
+
+from google.appengine.ext import db
```

Comment:
Just a comment: you'll want to be able to describe what sort of database this is and what properties it has.